test/bench/ok/heap-32.drun-run.ok

```diff
@@ -1,5 +1,5 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
 debug.print: (50_227, +29_863_068, 769_000_085)
-debug.print: (50_070, +32_992_212, 830_427_314)
+debug.print: (50_070, +32_992_212, 830_427_376)
```

Comment:
Any idea why this has increased so much?

Reply:
Oh, weird maybe it hasn't changed. When I was commenting on the recent changes, the diff was more like 10%

Reply:
It is just a few cycles. Not a big deal.